src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $transition-duration;
```

Comment:
I see your hover effect has not only `transform` state, but `color` as well. 
Add this `transition-property`